.pre-commit-config.yaml

```diff
@@ -9,17 +9,6 @@ repos:
     rev: v3.19.0
     hooks:
       - id: pyupgrade
-  - repo: https://github.com/astral-sh/ruff-pre-commit
-    # Ruff version.
-    rev: v0.7.0
-    hooks:
-      # Run the linter.
-      - id: ruff
-        types_or: [python, pyi, jupyter]
-        args: [--fix]
-      # Run the formatter.
-      - id: ruff-format
-        types_or: [python, pyi, jupyter]
```

Comment:
Note that I plan to create a subissue in #589 to revert the commit for this change.